prometheus-exporters/install-exporter-amd64.sh

```diff
@@ -353,6 +369,12 @@ default:
 EOF
   fi
 
+  if [ "$EXPORTER" == "postgres" ]; then
+    cat << EOF > /etc/opsverse/exporters/postgres/postgres_exporter.env
+DATA_SOURCE_NAME="postgresql://postgres:postgres123@localhost:5432/?sslmode=disable"
```

Comment:
Since this is likely a random credentials, why don't we just mention `postgres_user:pistgres_password` . This way it will be easy to refer to this in our docs and it will be easy for users to locate what they need to change.

Reply:
Okay Arul
I have made the changes.